daemons/attrd/pacemaker-attrd.h

```diff
@@ -78,19 +78,13 @@ int attrd_expand_value(const char *value, const char *old_value);
 
 /* regular expression to clear failure of all operations for one resource
  * (format takes resource name)
- *
- * @COMPAT attributes set < 1.1.17:
```

Comment:
fyi theoretically if someone has done rolling upgrades from <1.1.17 to >2.0.0 and never cleaned resources that whole time, some of these could still be around. i think that's an acceptable risk :)

Reply:
That sucks but yeah we ought to drop this at some point, and now is as good a time as any. At least those stale attributes will go away eventually, in that edge case -- at reboot if nothing else.